src/solver/type_getters.jl

```diff
@@ -146,26 +146,7 @@ function get_spaces(parsed_args, params, comms_ctx)
     elseif topography == "NoWarp"
         warp_function = nothing
     elseif topography == "Earth"
-        data_path = joinpath(topo_elev_dataset_path(), "ETOPO1_coarse.nc")
-        array_type = ClimaComms.array_type(comms_ctx.device)
-        earth_spline = NCDatasets.NCDataset(data_path) do data
-            zlevels = Array(data["elevation"])
-            lon = Array(data["longitude"])
-            lat = Array(data["latitude"])
-            # Apply Smoothing
-            smooth_degree = Int(parsed_args["smoothing_order"])
-            esmth = CA.gaussian_smooth(zlevels, smooth_degree)
-            Adapt.adapt(
-                array_type,
-                linear_interpolation(
-                    (lon, lat),
-                    esmth,
-                    extrapolation_bc = (Periodic(), Flat()),
-                ),
-            )
-        end
-        @info "Generated interpolation stencil"
-        warp_function = generate_topography_warp(earth_spline)
+        warp_function = nothing
```

Comment:
Is there a reason for setting warp_function to nothing for Earth topography, and using parsed_args["topography"] to set z_surface in `make_hybrid_spaces`? I find it a bit confusing that warp_function is nothing for Earth topography.

Reply:
The previous iteration was an attempt to keep this a non-breaking change; I've updated the `make_hybrid_space` function to use `SpaceVaryingInputs` everywhere which simplifies some of these branches.